lib/panoptes-subject-uploader.iced

```diff
@@ -141,38 +148,37 @@ for file in args._
       subject.metadata = metadata
       subject.locations = []
       subject.links = project: args.project
-
-      # find a https urls for the row
       imageURLs = findImagesURLs metadata
-      if imageURLs.length is 0
-        log "!!! Cannot find a https url for row #{i + 1}"
-        break
+      
+      if imageURLs.length == 0
+        console.error "!!! Couldn't find an media files for row #{i + 1}"
```

Comment:
For consistency, can we use the `log` function that was written? I'll be adding more functionality to it for #8